src/main/java/redis/clients/jedis/Client.java

```diff
@@ -39,9 +39,13 @@ public void get(final String key) {
     get(SafeEncoder.encode(key));
   }
 
-  public void exists(final String key) {
-    exists(SafeEncoder.encode(key));
-  }
+  public void exists(final String... keys) {
```

Comment:
@charsyam 
We're breaking binary compatibility now. 
Could you leave origin method as is but adding @Deprecated?